### PR TITLE
Rename folders of modules, add Logger module and configurations

### PR DESCRIPTION
**Pull request by snripa**

### CODE REVIEW

File: Logger/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    </dependencies>
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+            </resource>
+        </resources>
+    </build>
+
+</project>
```
Comment (artem-v) [inline Logger/pom.xml]: Could you please add a new line/

File: Logger/pom.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (artem-v) [inline Logger/pom.xml]: Looks like this files is not formatted with proper indent size.

Comment (snripa) [reply Logger/pom.xml]: it is(

File: pom.xml [hunk 3]
```diff
@@ -78,12 +79,13 @@
   </properties>
 
   <modules>
-    <module>Http</module>
+    <module>Logger</module>
```
Comment (ronenhamias) [inline pom.xml]: why folders starts with camel case?
its not anywhere in any other scalecube project.

Comment (snripa) [reply pom.xml]: please consult with @artem-v , dont want to change it in this PR

File: Logger/pom.xml (hunk 2 above)
Comment (artem-v) [reply Logger/pom.xml]: Let's format ! )

**Pull request merged**